src/tools/schema2base.ts

```diff
@@ -29,18 +30,20 @@ export abstract class Schema2Base {
   }
 
   private async processFile(src: string) {
+    Utils.init('../..');
     const outName = this.opts.outfile || this.outputName(path.basename(src));
     const outPath = path.join(this.opts.outdir, outName);
     console.log(outPath);
     if (this.opts.update && fs.existsSync(outPath) && fs.statSync(outPath).mtimeMs > fs.statSync(src).mtimeMs) {
       return;
     }
 
-    const contents = fs.readFileSync(src, 'utf-8');
-    const manifest = await Manifest.parse(contents);
+    const srcPath = `${process.cwd()}/${src}`;
+    const manifest = await Utils.parse(`import '${srcPath}'`);
 
     // Collect declared schemas along with any inlined in particle connections.
-    const schemas: Dictionary<Schema> = {...manifest.schemas};
+    const schemas: Dictionary<Schema> = {};
+    manifest.allSchemas.forEach(schema => schemas[schema.name] = schema);    
```

Comment:
@shaper Is it worth it to mangle the keys in the dictionary so no schema get overriden? 
something like (but cleaner): 
```
const schemas: Dictionary<Schema> = manifest.allSchemas.reduce((schemas, schema) => {
 let key = schema.name;
    while (key in schemas) {
      key += '_';
    }
    schemas[key] = schema;
    return schemas;
}, {}); 
```

Reply:
I think this issue can be dealt with in a separate PR.